lib/website_templates/adf_diag.css

```diff
@@ -95,56 +120,110 @@ h3 {
 }
 
 img {
-  max-width:75%;
-  height:auto;
+  max-width: 75%;
+  height: auto;
+}
+
+table.big-table{
+  border-collapse: collapse;
+  border-radius: 10px;
+  background-color: #012169;
 }
 
 table.big-table ul,ol,dl,p,th {
   font-size: 1.0rem;
   font-family: Tahoma, Geneva, Verdana, sans-serif;
-  color: rgb(67, 67, 67);
+  color: #D9D9D6;
   text-align: left;
 }
 table.big-table td {
   font-family: Tahoma, Geneva, Verdana, sans-serif;
-  padding-left: 10px;
+  /* background-color: #012169; */
```

Comment:
I would just delete this line rather than comment it out.

Reply:
Done!